include/zephyr/bluetooth/gatt.h

```diff
@@ -1657,15 +1657,23 @@ struct bt_gatt_discover_params {
 		struct {
 			/** Include service attribute declaration handle */
 			uint16_t attr_handle;
-			/** Included service start handle */
+			/** Starting attribute handle for included service */
 			uint16_t start_handle;
-			/** Included service end handle */
+			/** Ending attribute handle for included service */
 			uint16_t end_handle;
 		} _included;
-		/** Discover start handle */
+		/** Starting attribute handle to begin discovery
+		 * 
+		 * @note When discovery begins in @ref bt_conn_cb.connected callback this is 
+		 * typically set to @ref BT_ATT_FIRST_ATTRIBUTE_HANDLE
+		 */
```

Comment:
IIRC, this value is also incremented by the stack for each call to the callback. 

I suggest we also document that behavior